scripts/vmbuild.rb

```diff
@@ -30,19 +31,6 @@
   build_label = "#{cli_options[:type]}-#{cli_options[:manageiq_ref]}"
 end
 
-BUILD_BASE          = Pathname.new("/build")
-GPG_DIR             = Pathname.new("/root/.gnupg")
-CFG_DIR             = BUILD_BASE.join("config")
-FILESHARE_DIR       = BUILD_BASE.join("fileshare")
-REFS_DIR            = BUILD_BASE.join("references")
-IMGFAC_DIR          = BUILD_BASE.join("imagefactory")
-IMGFAC_CONF         = CFG_DIR.join("imagefactory.conf")
-STORAGE_DIR         = BUILD_BASE.join("storage")
-
-FILE_SERVER         = ENV["BUILD_FILE_SERVER"]             # SSH Server to host files
-FILE_SERVER_ACCOUNT = ENV["BUILD_FILE_SERVER_ACCOUNT"]     # Account to SSH as
-FILE_SERVER_BASE    = Pathname.new(ENV["BUILD_FILE_SERVER_BASE"] || ".") # Subdirectory of Account where to store builds
-
```

Comment:
@geertj from a reviewer perspective, I have to scroll up and down to see if this code was just moved or moved and modified.  It looks identical, but moved.  Can you do the move to a new file in a single commit so I can review commit by commit and not have to keep all of these changes in my head?